docs/content/en/sentry/hooks

```diff
@@ -0,0 +1,24 @@
+---
+title: Report Pre-processing (Hooks)
```

Comment:
I'd prefer non-titleCase as this casing looks a bit random.

`Report pre-processing (hooks)`